docs/devices/cloud-protocol/SensorDataAndConfiguration.rst

```diff
@@ -49,12 +49,18 @@ See the `ncellmeas JSON document <./ncellmeas.json>`_ for an example report.
 Other messages
 --------------
 
-If supported through enabling :ref:`aws-nrf-cloud-location-services`, the device can request A-GPS data through MQTT.
-The message published by the device is described in detail in the `A-GPS JSON schema file <./agps-request.schema.json>`_.
-See the `A-GPS request JSON document <./agps-request.json>`_ for an example request.
-The cloud publishes the requested types as binary to the ``<device Id>/agps`` topic.
+If supported through enabling :ref:`aws-nrf-cloud-location-services`, the device can request A-GPS and P-GPS data through MQTT.
+
+The A-GPS message published by the device is described in detail in the `A-GPS JSON schema file <./agps-request.schema.json>`_. 
+See the `A-GPS request JSON document <./agps-request.json>`_ for an example request. 
+The cloud publishes the requested types as binary to the ``<device Id>/agps`` topic. 
 If the device requests ephemerides (type 2), this message is too large to combine with other types and will be published in a separate message.
 
+The P-GPS message published by the device is described in the `P-GPS request JSON schema file <./pgps-request.schema.json>`_. 
+See the `P-GPS request JSON document <./pgps-request.json>`_ for an example request. 
+The cloud publishes a link to the data to the ``<device Id>/pgps`` topic in the format described in detail in the `P-GPS response JSON schema file <./pgps-response.schema.json>`_.
+See the `P-GPS response JSON document <./pgps-response.json>`_ for an example response.
+The device then can use HTTP(s) to download the P-GPS data.
```

Comment:
```suggestion
The device can then use HTTP or HTTPS to download the P-GPS data.
```
Alternatively, "use one or more HTTPs to download" if parenthetical s means plural instead of HTTPS.